### ISSUE TITLE
the form token is not valid - fresh install

### ISSUE THREAD

**Issue by shaunmitchellve** [drakkan/sftpgo#975]
Hello,

I have just created a new installation of SFTGO using the latest container `drakkan/sftpgo:v2.3` (SFTGO Version: 2.3.4) and the only configuration change I've made is to setup the postgresql data provider.

I can see the tables in the postgresql database and the container stars successfully. When go `http://URL/admin/login` after setting my default admin / password I get the error `the form token is not valid`. I did also get this error when trying to setup the default admin / password but it seems to have gotten through.

I haven't setup any certs, etc..

I'm attempting to run SFTGO on a Kubernetes cluster with 3 replicas and an ingress through a cloud load balancer.

**Comment by drakkan** [comment]
You have to setup SFTPGo to work in cluster mode. No time/motivation for step-by-step instructions, sorry.

Closing according to the [support policy](https://github.com/drakkan/sftpgo#support-policy)

**Comment by yanc0** [comment]
I have the same problem, even if I only have one running instance of sftpgo behind a Load Balancer. I'm trying to tweak the config file to find the right params like:

```
"client_ip_proxy_header": "X-Forwarded-For",
"client_ip_header_depth": 1,
```

**Comment by drakkan** [comment]
@shaunmitchellve thanks for sharing. 
@yanc0 in your case I guess you need to set the allowed proxies and you don't need a common signing passphrase if you have a single SFTPGo instance

**Comment by yanc0** [comment]
Thank you all, It worked on GCP with the Load Balancer.

**Comment by bkanuka** [comment]
Hey @shaunmitchellve - where did those "magic numbers"  `130.211.0.0/22,35.191.0.0/16` come from?  I can see in logs that requests _are_ coming from those ip's, but why? Are all GCP LB's in that range?

**Comment by butterthon-dev** [comment]
@shaunmitchellve 
> name: SFTPGO_HTTPD__BINDINGS__0__CLIENT_IP_PROXY_HEADER
value: "X-Forwarded-For"
name: SFTPGO_HTTPD__BINDINGS__0__CLIENT_IP_HEADER_DEPTH
value: "0"
name: SFTPGO_HTTPD__BINDINGS__0__PROXY_ALLOWED
value: "130.211.0.0/22,35.191.0.0/16"

thanks for sharing.
I was suffering from the same problem, but you have freed me :)

**Comment by butterthon-dev** [comment]
Hi @drakkan.

>name: SFTPGO_HTTPD__BINDINGS__0__CLIENT_IP_PROXY_HEADER
value: "X-Forwarded-For"
name: SFTPGO_HTTPD__BINDINGS__0__CLIENT_IP_HEADER_DEPTH
value: "0"
name: SFTPGO_HTTPD__BINDINGS__0__PROXY_ALLOWED
value: "130.211.0.0/22,35.191.0.0/16"

Thank you for providing configuration values for a variety of use cases！

I am Japanese and use a translation tool, so my English may be wrong.
Sorry.

**Comment by nhuray** [comment]
Hi community,

I have followed @shaunmitchellve [comment](https://github.com/drakkan/sftpgo/issues/975#issuecomment-1239428031) and it was partially working for me on GKE. Sometimes I was able to connect to the Web Admin, sometimes I had the message "Form token is not valid".

I finally solved the issue setting the `token_validation` documented [here](https://docs.sftpgo.com/latest/config-file/#http-server) in my Kubernetes manifest:

```yaml
  - name: SFTPGO_HTTPD__TOKEN_VALIDATION
    value: "1"
```

Not sure it's a good idea to disable that validation on production, but for my use case it's acceptable.
